Example/SwiftPrettyPrintExample/Source/AppDelegate.swift

```diff
@@ -19,6 +19,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // --------
         // Example
         // --------
+
+        *"Example"*
```

Comment:
I think that operator-based API make confusing for this OSS user.
This is example project, so I think most important things is that it is easy for users to understand.

I suggest to change that use simple function instead of operator-based API.

For example:

```swift
printSection("Example")
```

Reply:
> most important things is that it is easy for users to understand.

I think so too.

I'll rewrite the part based on this suggestion.

Reply:
I have reflected the point at https://github.com/YusukeHosonuma/SwiftPrettyPrint/pull/154/commits/3e8c5db9aa8c1c28a1256ad1e23e7ebc944f2e8d .